README.md

```diff
@@ -1,2 +1,56 @@
-# todoList
-This project base on the daily task list
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List (Structure)
+
+> This project base on the struction of add/remove daily task list
+
+![Screenshot (74)](https://user-images.githubusercontent.com/61976324/149188620-4fc7efe1-9122-4070-a4c6-116d9a2d6082.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages: HTML, CSS and Javascript
+- Frameworks: NO
+- Technologies used: Github and git flow, VS code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+### Prerequisites: Set up webpack
+
+### Setup: Vs Code 
```

Comment:
- [ ] (Required) It would be nice to include setup configuration for this project. That way anybody that sees your project will be able to follow your guide and be able to run a local copy of your project on their machine 👍.

Reply:
Thanks 👍. You don't approve this just because of readme. I don't feel content with this. 😎

Reply:
@oyelakinG9 we are trying to make sure you document your project professionally, which is one the requirements in your project in the dashboard. Our concern is to make sure that you have a great readme file that could impress any recruiter. We are here to support one another.

Thank you.

Reply:
I've updated it @HENRYKC24 . You can check for new update. 👍